docs/source/data-types/data-types.md

```
@@ -0,0 +1,158 @@
+await client.execute(
+  "INSERT INTO t (id, val) VALUES (?, ?)",
+  [id, someValue],
+  { hints: ["uuid", "int"] }
+);
+```
+
+### Text Representation
+
+Pass a string describing the CQL type:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)